tests/integration_tests/test_version_command.py

```diff
@@ -32,3 +33,44 @@ def or_fail():
 
     await helpers.get_test_channel().send("!version")
     await helpers.wait_for(version_to_appear, or_fail)
+
+
+@helpers.threaded_async
+async def test_returns_error_on_missing_role():
+    """
+    This function makes sure that members who do not have the "Defender" role can't use the !version command.
+    The function removes the "Defender" role from DofDevBot, then it makes the bot use the command !version as a user.
+    The bot shouldn't be able to have the version as result for his command due to his role not being the one required.
+    At the end the "Defender" role is added back to DofDevBot.
```

Comment:
1. Replace "make sure" with "tests" -> it's a test after all!
2. Remove "from DofDevBot" to make it shorter and avoid referencing our development application
3. "At the end the" should have a comma between "end" and "the". No need for mention of the bot after "back"
4.  Replace "have the version" with "list the version"
5. Add some spacing - the first line of docs should always explain the function as generic as possible. Next lines can go into details.